lib/aggregators/sum.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const printers = require('../printers');
+const Table = require('../table');
+function calculateSum (table1, table2) {
```

Comment:
it could, but I think the intent is clearer when reading `function doThis () {}` than `const doThis = () => {}`. I tend to mostly use arrow functions for anonymous functions which are defined exactly where they are called and where you deliberately don't want to change the value of `this`